components/ArticleReplyFeedbackControl/ReasonsDisplay.js

```diff
@@ -95,9 +95,21 @@ function ReasonsDisplay({ articleReply, onSizeChange = () => {} }) {
     if (onSizeChange) return onSizeChange();
   }, [tab, onSizeChange]);
 
+  const IsEmptyComment = comment => comment === '' || comment === null;
```

Comment:
nit: please help moving this helper function to outside of the component, so that the browser don't create this function on every render.
Also, we can use camelCase instead of PascalCase, which is mostly reserved for React components.